### DISCUSSION TITLE
TEALS SNAP 2.3.pptx Is that a good example?

### DISCUSSION THREAD

**Discussion by jdonwells** [TEALSK12/introduction-to-computer-science#418] [Ideas]
The example code we show them on slide 7 cannot be fixed with what they know now. Shouldn't we show them an example that doesn't work followed by the correct way to do it? We can't do that with the given example, we just leave them hanging until the next unit.

The notes on that slide say as much.

Instead, we could show them this:
![Screenshot 2022-10-03 4 38 31 PM](https://user-images.githubusercontent.com/5577101/193678755-e99e695f-b74f-4242-a031-3e77d635b8e8.png)

With a solution that looks like this:
![Screenshot 2022-10-03 4 40 47 PM](https://user-images.githubusercontent.com/5577101/193678805-a20142f5-63de-443b-b8cd-6a02a42f2143.png)

**Comment by aspiece** [comment]
Completely agree. I didn't like it when I presented it this week. I plan to update on my slides. Should be fixed.
